### PR TITLE
Files as argument

### PR DESCRIPTION
**Pull request by JelleHissink**
* added files verb,
* updated readme
* allow to specify output level (needed verbose to list all files that are processed to verify myself ;-) )
* did a start for sass processing options (outputstyle)

### CODE REVIEW

File: src/LibSassBuilder/FilesOptions.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using CommandLine;
+using System.Collections.Generic;
+
+namespace LibSassBuilder
+{
+	[Verb("files")]
+	public class FilesOptions : GenericOptions
+	{
+		[Value(0, Required = true, HelpText = "Directory in which to run. Defaults to current directory.")]
```
Comment (johan-v-r) [inline src/LibSassBuilder/FilesOptions.cs:9]: Could you please give a bit more descriptive `HelpText` here.

File: src/LibSassBuilder/FilesOptions.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+		public IEnumerable<string> Files { get; set; } 
+
+		[Option('e', "exclude", Required = false, HelpText = "Specify explicit directories to exclude. Overrides the default.", Default = new[] { "bin", "obj", "logs", "node_modules" })]
+		public IEnumerable<string> ExcludedDirectories { get; set; }
```
Comment (johan-v-r) [inline src/LibSassBuilder/FilesOptions.cs:13]: This can be removed as not used.

**Review by johan-v-r — CHANGES_REQUESTED** [review]
Awesome I like where this is heading 👍

File: src/LibSassBuilder/FilesOptions.cs (hunk 1 above)
Comment (JelleHissink) [reply src/LibSassBuilder/FilesOptions.cs:9]: haha, that was your original text ;-)

Comment (JelleHissink) [reply src/LibSassBuilder/FilesOptions.cs:9]: Sry, I'll check... got my view of your comment a little messed up ;-)

Comment (JelleHissink) [reply src/LibSassBuilder/FilesOptions.cs]: Done

File: src/LibSassBuilder/FilesOptions.cs (hunk 2 above)
Comment (JelleHissink) [reply src/LibSassBuilder/FilesOptions.cs]: Done

**Pull request merged**